static/styles/_components/_entity.scss

```diff
@@ -75,6 +86,47 @@
   }
 }
 
+.time-period {
+  width: 100%;
+  float: left;
+  padding: 1rem 0;
+  label {
+    width: 50%;
+    margin-right: 1rem;
+    display: inline-block;
+  }
+
+  select,
+  .chosen-container {
+    width: 50% !important;
```

Comment:
Ok, I'll be able to clean this up with Chosenjs gone. I'm going to wait to style until that's all merged though.